TwilioVerifyDemo/NotificationExtension/NotificationService.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  NotificationService.swift
+//  TwilioVerifyDemo
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import UserNotifications
+import TwilioVerifySDK
+import TwilioVerifyDemoCache
+
+class NotificationService: UNNotificationServiceExtension {
+  
+  var contentHandler: ((UNNotificationContent) -> Void)?
+  var bestAttemptContent: UNMutableNotificationContent?
```

Comment:
We should rename this variable `bestAttemptContent` to something like `lastMutableNotificationContent`.

Reply:
Thanks! but I left this as the default implementation, this was created with the extension itself, I haven't touch it. I though it would be better to left this as most default possible so you can improve it in your ticket. I only created this extension to test the cache. Same with the title behavior.